cucumber/base/psmisc/psmisc.buildinfo

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Copyright 2018 Scott Court and Zach Jorgensen
```

Comment:
Please put each author on a separate line with the newest on the bottom and please keep the original years intact. I.e.:
`# Copyright 2016 Scott Court`
`# Copyright 2018 Zach Jorgensen`